msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -9,14 +9,17 @@ ms.topic: overview
 
 # Customize your Teams app
 
-## Enable your Microsoft Teams app to be customized
+Microsoft Teams lets admins customize Teams app to enhance store experience and adhere to their organization's branding. An app developer can allow their app to be customized by a Teams admin. For more information, see
+[Customize apps in the Teams admin center](/MicrosoftTeams/customize-apps).
+
+## Enable customization for your Microsoft Teams app
```

Comment:
Redundant H2.

Reply:
Not a redundant H2 anymore, given that (for now) we are covering 2 topics in the same article.